lib/codelog/command/regenerate.rb

```diff
@@ -0,0 +1,13 @@
+module Codelog
+  module Command
+    class Regenerate
+      def self.run
+        Codelog::Command::Regenerate.new.run
+      end
+
+      def run
+        Codelog::Command::Step::Changelog.run
```

Comment:
Would be nice to give the user a message telling that the changelog file was regenerated

Reply:
Done 👍